cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -733,28 +756,25 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                     expect($("#due_date").val()).toBe('7/10/2014');
                     expect($("#grading_type").val()).toBe('Lab');
                     expect($("#staff_lock").is(":checked")).toBe(true);
-                    expect($("#id_timed_examination").is(":checked")).toBe(true);
-                    expect($("#id_exam_proctoring").is(":checked")).toBe(true);
-                    expect($("#is_practice_exam").is(":checked")).toBe(false);
+                    expect($("#id_timed_exam").is(":checked")).toBe(false);
```

Comment:
Because of the notes above, I think this test is just testing the code in setModalTimedExaminationPreferenceValues, not course_outline_modals.js.